webapp/src/components/viewTitle.tsx

```diff
@@ -64,6 +64,7 @@ const ViewTitle = React.memo((props: Props) => {
                         }}
                         icon={<ShowIcon/>}
                     >
+                        &nbsp;
```

Comment:
I don't love the use of `&nbsp;` here, it remove certain control of the styling, do you mind to add a `.ViewTitle .add-buttons .Icon` css that add a `margin-right`, that way we can make more fine grain styling of this.

Reply:
by the way, the right place would be the file `webapp/src/components/viewTitle.scss`